nautobot/utilities/tables.py

```diff
@@ -365,11 +365,11 @@ def render(self, record, bound_column, value):
         if value is None:
             return self.default
 
+        template = ""
         if self.customfield.type == CustomFieldTypeChoices.TYPE_BOOLEAN:
             template = render_boolean(value)
         elif self.customfield.type == CustomFieldTypeChoices.TYPE_MULTISELECT:
             if value:
-                template = ""
                 for v in value:
                     template += format_html('<span class="label label-default">{}</span> ', v)
```

Comment:
I wonder - if I'm reading this correctly, this means that for a multiselect with no selections made, we get an empty `template`. I think it might be more correct to do:

```python
        elif self.customfield.type == CustomFieldTypeChoices.TYPE_MULTISELECT:
            if not value:
                return self.default
            for v in value:
                template += format_html('<span class="label label-default">{}</span> ', v)
```